packages/components/package.json

```diff
@@ -99,6 +99,7 @@
     "react-dom": "~16.8.6",
     "react-dropzone": "^9.0.0",
     "react-pose": "^4.0.8",
+    "react-tiny-popover": "^3.4.5",
```

Comment:
Should we include `react-tiny-popover` to externals in `webpack` config?

Reply:
If you'd like to do that, please note that you should add it as a `peerDependency` and then add it as a dependency in Core.

Reply:
Added